src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,232 +1,176 @@
 
-Any portion of this procedure may be waived and/or modified with written permission (_via Discord or Github_) from a Lead Maintainer, Project Manager or Wizard.
-
-Failure to follow this procedure can result in disciplinary action.
-
-## PR Requirements
-All PRs must adhere to the [code conventions](../../general-development/codebase-info/conventions.md).
-
-All PRs must be triaged, as per the [triage procedure](triage-procedure.md) guidelines.
-Upon actioning on a PR (merging, closing, reviewing), if it is not triaged, the person actioning must triage the PR.
-
-All PRs must not conflict with [core design principles](../../space-station-14/core-design/design-principles.md). 
-Additionally, the PR author should describe how the changes they made fit into the game.
-Sufficiently large feature PRs may also warrant a **Design Doc** detailing the broader scope and purpose of their changes.
+Pull requests targeting the `staging` or `stable` branch are subject to the [Hotfix Procedure](hotfix-procedure.md).
 
-All PRs must completely fill out the Github PR Template to a satisfactory level.
-This includes filling out all applicable sections and attaching media when necessary.
-
-### Breaking Changes
-
-Breaking changes are defined as one or more of the following:
-- Modifying a public API
-- Moving code into a different namespace
-- Changing prototype IDs (*even if migrations are present*)
-
-All PRs that include **breaking changes** should also include a section with a summary of the changes as well as instructions on how to fix issues that may come from them.
-
-After the PR is merged, a maintainer should make a post in the [breaking changes](https://forum.spacestation14.com/c/development/breaking-changes/70) section of the forum using the template.
-This consists only of the breaking changes section from the PR itself as well as a link to the PR.
+Any portion of this procedure may be waived and/or modified with written permission (_via Discord or Github_) from a Lead Maintainer, Project Manager or Wizard.
 
-## Decision Policy
-```admonish info "Exceptions on Maintainer Workgroups"
-[Maintainer Workgroups](maintainer-workgroup-policy.md) have exceptions to policy regarding decisions and discussions.
-This is intended to help expedite pull requests and maintain a design direction, by directing applicable pull requests to their relevant maintainer workgroups.
+```admonish note
+Note that these procedures are ultimately flexible, however Maintainers that deviate from this policy are expected to provide concrete and valid reasoning when doing so.
+Getting one or more other maintainer(s) to agree with your deviation is advisable.
 
-Exceptions to policy are listed where applicable.
+Note that all pull requests that are scheduled for release to `stable` are reviewed during _Stable Review_ in maintainer meetings.
```

Comment:
This is only the case for Feature PRs or sometimes particularly notable refactors, and of those, nowadays we generally only review the ones that had comments on them (although recently, we had deviated from that sometimes, when we had a quiet week so it took less time to go through them all)

If the intent is to communicate that they can still be changed or reverted, as noted above, then it could just say `are reviewed before or during _Stable Review_` which is a small change, but we probably want the policy to accurately say what we are doing